src/eko/anomalous_dimensions/as1aem1.py

```diff
@@ -360,7 +349,7 @@ def gamma_nsm(N, nf, sx):
         + 8.0 / (N + N**2) * S2p1h
         + S1
         * (
-            -16 * (3 / N**2 - 3 / (1 + N) ** 2 - 2 * zeta2)
+            16 * (-1 / N**2 + 1 / (1 + N) ** 2 + 2 * zeta2)
```

Comment:
I guess the rescaling by a factor of 3 is intentionally?

Reply:
Yes it is. I forgot it when I copied from mathematica to python. In fact if you put it back all the sum rules are broken